file_writer_control/WorkerFinder.py

```diff
@@ -125,5 +126,10 @@ class WorkerFinder(WorkerFinderBase):
     def __init__(self, command_topic_url: str):
         temp_cmd_ch = CommandChannel(command_topic_url)
         command_url = KafkaTopicUrl(command_topic_url)
-        temp_producer = KafkaProducer(bootstrap_servers=[command_url.host_port])
+        try:
+            temp_producer = KafkaProducer(bootstrap_servers=[command_url.host_port])
```

Comment:
Unnecessary empty line :)

Reply:
Hmm. May be there is no empty line? In review view there was :)

Reply:
I can see no empty line there in the actual source file